features/0023-did-exchange/README.md

```diff
@@ -313,13 +317,13 @@ When the message is sent, the _responder_ are now in the `response-sent` state.
 
 #### Response Processing
 
-When the _invitee_ receives the `response` message, they will verify the `change_sig` provided. After validation, they will update their wallet with the new information. If the endpoint was changed, they may wish to execute a Trust Ping to verify that new endpoint.
```

Comment:
This is the first and only mention of `change_sig`: what is it?

Reply:
I removed the reference and attempted to clarify above. Basically, if the responder had initiated the connection via an OOB invitation or a public DID and then switched to a peer DID, the DIDDoc signature should use the DID/key from the OOB or public DID.